File: green/scalr-utils/facebook/updateRefhandle.py
import facebook
import datetime
import sys

# fb = facebook.Facebook('bd100b49340effa90332cdfbe6e85659','b150ebbad9cdff51269248dbe4b9683d')
api_key = 'bd100b49340effa90332cdfbe6e85659'
secret_key = 'b150ebbad9cdff51269248dbe4b9683d'
fb = facebook.Facebook(api_key,secret_key)
stamp = datetime.datetime.now()
fb.fbml.setRefHandle('myhandle','Last Updated at %s'%stamp)
sys.exit(0)


fb.auth.createToken()
fb.login()
print "Press Enter When logged in."
sys.stdin.readline()  # sleep(100)

session = fb.auth.getSession()
print session


print "Get User Info"
print fb.users.getInfo([fb.uid], ['name', 'birthday'])

print "Setting Profile Info:"
fb.profile.setFBML('',fb.uid,'Box','','','Wall and Info')

File: trunk/green/scalr-utils/facebook/updateRefhandle.php
<?php
require_once 'facebook.php';

$now = time();
$stamp = date('Y-m-d',$now).'T'.date('H:i:s',$now).'Z';
$fbml = "Last Updated at ".$stamp;
echo "Update Facebook iMetrical refHandle:myhandle to:\n";
echo "  ".$fbml."\n";

// fb.fbml.setRefHandle('myhandle','Last Updated at %s'%stamp)

$appapikey = 'bd100b49340effa90332cdfbe6e85659';
$appsecret = 'b150ebbad9cdff51269248dbe4b9683d';
$facebook = new Facebook($appapikey, $appsecret);

$facebook->api_client->call_method('facebook.Fbml.setRefHandle', array( 'handle' => 'myhandle', 'fbml' => $fbml,) ); 

echo "Done\n"
?>


